test/nonregression/testing_tools.py

```diff
@@ -8,6 +8,20 @@
 import pandas as pd
 
 
+def configure_paths(
+    base_dir: PathLike,
+    tmp_path: PathLike,
+    name: str,
+) -> Tuple[PathLike, PathLike, PathLike]:
```

Comment:
You are gonna have a type alignment error since `os.PathLike` does not implement path concat with `/`. You'd need to either use `Path` or `PurePath` all the way, or do explicit conversion within the function.

```suggestion
    base_dir: Path,
    tmp_path: Path,
    name: str,
) -> Tuple[Path, Path, Path]:
```

Considering this is just a utility function, `os.PathLike` is not much interesting to be honest.

Reply:
Thanks! 👍  I saw it but forgot to fix it before asking for a review.